docs/assets/css/sidebar.less

```diff
@@ -50,12 +45,21 @@
 
 // Mobile size.
 .respond-to-max( @bp-xs-max, {
-    .ds-nav .h4 {
-        margin-bottom: 0;
+    
+    .ds-nav a {
+        border-style: solid;
+        border-top-width: 0;
     }
 
-    .ds-nav-2 {
+    .ds-nav-2 .ds-nav-3 {
```

Comment:
Are you trying to target both `ds-nav-2` and `ds-nav-3`? If it's both of them, you'll need a comma 

```
.ds-nav-2,
.ds-nav-3 {
```

Reply:
Nice catch, thank you. Fixed.